_posts/2017-07-29-ruby-oop.md

```diff
@@ -0,0 +1,383 @@
+---
+layout: post
+author: z64
+title: Beginning OOP with Ruby
+date: 2017-07-29
+tags: ruby oop tutorial
+---
+
+This post attempts to guide a beginner through object-oriented progamming in Ruby, showing them the thought process behind both designing and interacting with a library, and problem solving in general with Ruby.
+
+To accomplish this, we will be solving a simple geometry problem:
```

Comment:
Probably reword this to "we will solve" to sound more like the intention of the post, rather than a plan to solve.